From 6b40f6c6ad5c4cee14d3bfb234bd848bbc5c947e Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@freescale.com>
Date: Mon, 9 Nov 2015 16:19:15 +0800
Subject: [PATCH 0662/5242] MLK-11841 dma: pxp: change 'CONFIG_PM_RUNTIME' to
 'CONFIG_PM'

commit  2d6f7ff5bf29f57c5f37c339d2bfa170025ac48b from
https://source.codeaurora.org/external/imx/linux-imx.git

The 'CONFIG_PM_RUNTIME' has been eliminated and all the users
should use 'CONFIG_PM' directly. So do this change for pxp-v2
and pxp-v3.

Signed-off-by: Fancy Fang <chen.fang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/dma/pxp/pxp_dma_v2.c |    2 +-
 drivers/dma/pxp/pxp_dma_v3.c |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/dma/pxp/pxp_dma_v2.c b/drivers/dma/pxp/pxp_dma_v2.c
index d9e75b8..0b19b9a 100644
--- a/drivers/dma/pxp/pxp_dma_v2.c
+++ b/drivers/dma/pxp/pxp_dma_v2.c
@@ -1818,7 +1818,7 @@ static int pxp_resume(struct device *dev)
 #define	pxp_resume	NULL
 #endif
 
-#ifdef CONFIG_PM_RUNTIME
+#ifdef CONFIG_PM
 static int pxp_runtime_suspend(struct device *dev)
 {
 	dev_dbg(dev, "pxp busfreq high release.\n");
diff --git a/drivers/dma/pxp/pxp_dma_v3.c b/drivers/dma/pxp/pxp_dma_v3.c
index 52c61cb..848285c8 100644
--- a/drivers/dma/pxp/pxp_dma_v3.c
+++ b/drivers/dma/pxp/pxp_dma_v3.c
@@ -4429,7 +4429,7 @@ static int pxp_resume(struct device *dev)
 #define	pxp_resume	NULL
 #endif
 
-#ifdef CONFIG_PM_RUNTIME
+#ifdef CONFIG_PM
 static int pxp_runtime_suspend(struct device *dev)
 {
 	dev_dbg(dev, "pxp busfreq high release.\n");
-- 
1.7.9.5

